screen/OrderData.xml.ftl

```diff
@@ -0,0 +1,167 @@
+                    <Company>
+                       <![CDATA[XX]]>
+                    </Company>
+                    -->
+                    <Address1>
+                       <![CDATA[${shipAddress.address1!}]]>
+                    </Address1>
+                    <Address2>${shipAddress.address2!}</Address2>
+                    <City>
+                       <![CDATA[${shipAddress.city!}]]>
+                    </City>
+                    <#if geo??>
+                    <State>
+                       <![CDATA[${geo.geoCodeAlpha2!}]]>
+                    </State>
+                    </#if>
+                    <PostalCode>
+                       <![CDATA[${shipAddress.postalCode!}]]>
+                    </PostalCode>
+                    <Country>
+                       <![CDATA[${shipAddress.countryGeoId!}]]>
+                    </Country>
+                    <Phone>
+                       <![CDATA[${shipAddress.contactNumber!}]]>
+                    </Phone>
+                 </ShipTo>
+              </Customer>
+              <Items>
+                 <#list items as item>
+                     <Item>
+                        <#assign dimension = ec.entity.find("mantle.product.ProductUomDimension").condition("productId",item.productId).condition("uomDimensionTypeId",'Weight').list()>
+                        <SKU>
+                           <![CDATA[${item.sku!}]]>
+                        </SKU>
+                        <Name>
+                           <![CDATA[${item.product_name!}]]>
+                        </Name>
+                        <#assign productContent = ec.entity.find("mantle.product.ProductContent").condition("productId",item.productId).condition("productContentTypeEnumId",'PcntImageLarge').list()>
+                        <#if !productContent.isEmpty()>
+                        <#assign url = ec.resource.getLocationReference(productContent[0].contentLocation)>
+                        <ImageUrl>
+                           <![CDATA[${url!}]]>
+                        </ImageUrl>
+                        </#if>
+                        <#if !dimension.isEmpty()>
+                        <#assign units = ec.entity.find("moqui.basic.Uom").condition("uomId",dimension[0].uomId).useCache(true).one()>
+                        <Weight>${dimension.value!}</Weight>
+                        <WeightUnits>${units.abbreviation!}</WeightUnits>
+                        </#if>
+                        <Quantity>${item.quantity!}</Quantity>
+                        <UnitPrice>${item.unitAmount!}</UnitPrice>
+                        <#--
+                        TODO: Provide value if exists
+                        <Location>
+                           <![CDATA[XX]]>
+                        </Location>
+                        -->
+                        <#assign featuresList = ec.service.sync().name("co.hotwax.oms.ProductServices.find#Products").parameter("productId", item.productId).call()>
```

Comment:
I think this will again get lots of redundant information about product. Find a better a way.